index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio Project</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h1 class="title">Welcome!</h1>
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
+    </nav>

```

Comment:
- [ ] Good job with your toolbar, however, there are some issues you need to address, please put all your toolbar elements in a `<header>` tag

Reply:
Thank you, I wrapped the nav inside a header